atd-vzv/src/views/nav/SideMapControlDateRange.js

```diff
@@ -189,27 +208,44 @@ const SideMapControlDateRange = ({ type }) => {
     border-radius: 4px;
     padding-left: 2px;
 
-    /* Center and enlarge picker reset button */
-    .reset-button {
-      position: relative;
-      top: 5px;
-      right: 2px;
-      width: 22px;
-      height: 22px;
-      cursor: pointer;
+    /* Center start and end date inputs */
+    [id^="start_date_"],
+    [id^="end_date_"] {
+      text-align: center;
```

Comment:
Ha! This is so clever ⚡👍

Reply:
Thanks - this was a tricky one to figure out. I'm still not 100% why these styles coming from the picker library were being overridden when deployed.